app/src/main/res/layout/fragment_asset_settings.xml

```diff
@@ -0,0 +1,311 @@
+
+            <View
+                android:id="@+id/divider_security"
+                style="@style/Divider"
+                android:layout_marginTop="17dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/text_security" />
+
+            <TextView
+                android:id="@+id/button_key"
+                android:layout_width="0dp"
+                android:layout_height="64dp"
+                android:background="@color/white"
+                android:clickable="true"
+                android:drawableEnd="@drawable/ic_arrow_right_grey"
+                android:focusable="true"
+                android:fontFamily="@font/montserrat"
+                android:foreground="?attr/selectableItemBackground"
+                android:gravity="center_vertical"
+                android:paddingEnd="10dp"
+                android:paddingStart="16dp"
+                android:text="@string/my_private_key"
+                android:textColor="@color/text_main_dark"
+                android:textSize="16sp"
+                android:textStyle="bold"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/divider_security" />
+
+            <View
+                android:id="@+id/divider_key"
+                style="@style/Divider"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/button_key" />
+
+            <TextView
+                android:id="@+id/notice_key"
+                style="@style/NotificationSettings"
+                android:text="@string/about_private_key"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/divider_key" />
+
+            <View
+                android:id="@+id/divider_notice"
+                style="@style/Divider"
+                android:layout_marginTop="20dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/notice_key" />
+
+            <TextView
+                android:id="@+id/button_delete"
+                android:layout_width="0dp"
+                android:layout_height="64dp"
+                android:background="@color/white"
+                android:clickable="true"
+                android:focusable="true"
+                android:fontFamily="@font/montserrat"
+                android:foreground="?attr/selectableItemBackground"
+                android:gravity="center"
+                android:text="@string/delete_wallet"
+                android:textColor="#e72f31"
```

Comment:
👎 
Every color must be exist in colors.xml. You can hardcode color values only in SVG xmls.